pkg/limayaml/defaults_test.go

```diff
@@ -282,11 +289,14 @@ func TestFillDefault(t *testing.T) {
 	// User-provided overrides should override user-provided config settings
 
 	o = LimaYAML{
-		Arch:    pointer.String(arch),
-		CPUType: pointer.String("host"),
-		CPUs:    pointer.Int(12),
-		Memory:  pointer.String("7GiB"),
-		Disk:    pointer.String("117GiB"),
+		Arch: pointer.String(arch),
+		CPUType: map[Arch]string{
+			AARCH64: "uber-arm",
```

Comment:
?

Reply:
Just random strings; they only need to be different from the lower priority settings, to verify that they are not being overwritten. When I wrote it, it felt like a funny fantasy-cpu name, but now I can't remember why I thought it was funny, so it probably isn't. I can change it if you want; suggestions welcome!